.azure/push.yml

```diff
@@ -0,0 +1,17 @@
+# The branch-protection policies mean that all code pushed to the relevant
+# branches must already have been made up-to-date and then tested during the PR
+# process, so testing the final push is largely just a quick spot-check.  A
+# merged PR only looks like an untested PR because of the "squash-and-merge"
+# policy; it creates a new commit object, even though the code state is exactly
+# the same (including being merged up with the base branch) as it was before.
+
+trigger:
+  branches:
+    include: ["main", "stable/*"]
+
+jobs:
+  - template: "templates/test-linux.yml"
+    parameters:
+      pythonVersions: ["3.9"]
```

Comment:
Is there a specific reason for 3.9?

Reply:
Not a particular one - it only needs to be one version of Python, and apparently that's the default in my head.  There's an argument to be made for 3.8 or 3.9 because they're not tested on PR sync, but equally you could argue that it should be one of the tested ones (3.7 or 3.10) because this is just a sanity check.

Reply:
If you do end up introducing a variables template imported by each pipeline, it might make it a little clearer that this is chosen purposefully, rather than in error. Maybe something like `PUSH_TEST_PYTHON_VERSION`.

Reply:
Done in 51884294f